src/index.html

```diff
@@ -13,6 +13,56 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <block class="block">
+      <container class="container">
```

Comment:
Need to fix it 
<img width="869" alt="image" src="https://github.com/user-attachments/assets/6956f992-fc96-4a0f-a7e4-473bf2f7e352">